content/blog/2021/10/02/MonitorTheSolver.adoc

```diff
@@ -0,0 +1,215 @@
+= Monitor the solver
+Christopher-Chianelli
+2021-10-02
+:page-interpolate: true
+:jbake-type: post
+:jbake-tags: monitoring, production
+
+It's 11pm on Friday evening. Everything was working fine -- until now.
+Suddenly services are failing left and right, and your boss want to know why.
+One tool in achieving a diagnosis is to use a monitoring system. Let's add monitoring to OptaPlanner and see if we can diagnose this issue.
```

Comment:
This paragraph is a very good start! It talks about the benefit for the user.

"the achieving sentence" I would drop. "add" I 'd replace by "enable" (it sounds like less effort and it is not a lot of effort).
```suggestion
Let's enable monitoring to the OptaPlanner nodes and see if we can diagnose this issue.
```